sa/sa_test.go

```diff
@@ -86,16 +86,18 @@ func createPendingAuthorization(t *testing.T, sa core.StorageAuthority, domain s
 	return ids.Ids[0]
 }
 
-func createFinalizedAuthorization(t *testing.T, sa core.StorageAuthority, domain string, exp time.Time, status string) int64 {
+func createFinalizedAuthorization(t *testing.T, sa core.StorageAuthority, domain string, exp time.Time, status string, atmpdAt time.Time) int64 {
```

Comment:
I'd rather spell out `attemptedAt` here. If you want to avoid shadowing it later when you convert to nanos, use `attemptedAtNanos`.

Reply:
Fixed. I went with attemptedAtInt to be similar to the expInt above.